### ISSUE TITLE
[A11y] BackButton and CloseButton should have appropriate labels.

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#110790]
Right now they only have tooltips and causes wrong announcement order. See b/234580483.
The native Android pronounce 'Back, button, double tap to activate'
The Flutter pronounce 'button, Back, double tap to activate'

If we do add labels to those buttons, we should make sure the announcement matches the native Android. 

This is also blocking internal customer to write integration test that relies on semantics label to locate the backbutton 
b/237459242

**Comment by chunhtai** [comment]
The native Android tooltip is also announce after the 'role' so  a button with label and tooltip is announce as
<label>, button, <tooltip>. It matches the flutter behavior. However, their backbutton seems to do a one off behavior. It has a tooltip but the tooltip is announced before the role.

**Comment by chunhtai** [comment]
It looks like they will set contentdescription for the button to match to tooltip, https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/widget/Toolbar.java;l=945;drc=04c8304ce850d08bc71b9c3695ad06a1e3078a5a;bpv=1;bpt=1.

They don't seem to add the semantics tooltip to the button directly. I am going to follow the same convention they do for back button.

**Comment by chunhtai** [comment]
another interesting fact is that if the content description is the same as tooltip text, talkback will actually skip the tooltip announcement, that explains why there won't be double announcement in the native back button.

**Comment by chunhtai** [comment]
Change is reverted to wait for customer migrate their code base

**Comment by chunhtai** [comment]
Lowering the priority as the customer no longer needs this fix. but it is still a bug as it doesn't match the native behavior.
